types/prosemirror-model/index.d.ts

```diff
@@ -13,17 +13,14 @@ export interface AnyObject {
 }
 
 export class ContentMatch {
-  matchNode(node: Node): ContentMatch | null | undefined;
   matchType(type: NodeType, attrs?: AnyObject, marks?: Mark[]): ContentMatch | null | undefined;
   matchFragment(fragment: Fragment, from?: number, to?: number): ContentMatch | boolean | null | undefined;
-  matchToEnd(fragment: Fragment, start?: number, end?: number): boolean;
-  validEnd(): boolean;
+  validEnd: boolean;
   fillBefore(after: Fragment, toEnd: boolean, startIndex?: number): Fragment | null | undefined;
-  allowsMark(markType: MarkType): boolean;
-  findWrapping(target: NodeType, targetAttrs?: AnyObject, targetMarks?: Mark[]): Array<{ type: NodeType, attrs: AnyObject }> | null | undefined;
-  findWrappingFor(node: Node): Array<{ type: NodeType, attrs: AnyObject }> | null | undefined;
+  findWrapping(target: NodeType, targetAttrs?: AnyObject, targetMarks?: Mark[]): NodeType[] | null | undefined;
```

Comment:
Only the first parameter `target` appears to be public API http://prosemirror.net/docs/ref/#model.ContentMatch.findWrapping. The description in the docs also implies it only returns `null` (rather than possibly `undefined`), but we'd have to check the implementation to be sure.

Reply:
Yes you are right. Is done